### DISCUSSION TITLE
Adjust the terms used in the information windows and the information bar at the bottom of the screen for more or less positive Morale (Awful, Treason, etc.) and for Luck [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2643] [Ideas]
In the original game, when the morale is equal to or higher than "Good" the information window and the information bar at the bottom of the screen indicates "Good Morale".
However, in the creature information window, the term used for troop morale is different and more related to the morale level, "Great" is then displayed.
It is the same for lower morale, and also for luck levels.
In fheroes2, it is the same.

I suggest replacing the term used in the hero screen for Morale and Luck information windows and for the information bar.

For Morale:
- "Blood" for +3 to morale.
- "Great" for +2 to morale.
- "Good" for +1 to morale.
- "Normal" for neutral morale.
- "Poor" for -1 to morale.
- "Awful" for -2 to morale.
- "Treason" for -3 to morale.

For Luck:
- "Irish" for +3 to luck.
- "Great" for +2 to luck.
- "Blood" for +1 to luck.
- "Normal" for neutral luck.
- "Bad" for -1 to luck.
- "Awful" for -2 to luck.
- "Cursed" for -3 to luck.

In fheroes2:

![Morale term change fh2 01](https://user-images.githubusercontent.com/43583994/105375956-82920200-5c09-11eb-9b82-433abf58e8f7.png)

![Morale term change fh2 02](https://user-images.githubusercontent.com/43583994/105375964-845bc580-5c09-11eb-8449-c3fbc4aee825.png)

![Morale term change fh2 03](https://user-images.githubusercontent.com/43583994/105375975-86258900-5c09-11eb-984f-f3927cdf2cd8.png)

**Comment by Branikolog** [comment]
I'd like to watch this improvement in the releases before 1.0, or at least in 1.0. )
I think this enhancement will not ruin the original gameplay, but can help newcomers to H2 game. (I hope up to 1.0 release lots of new people would like play our product.)
If it hasn't been already created, could you, please, create an "improvement" issue with a description of morale and luck types, @LeHerosInconnu ?

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> I'd like to watch this improvement in the releases before 1.0, or at least in 1.0. )
> I think this enhancement will not ruin the original gameplay, but can help newcomers to H2 game. (I hope up to 1.0 release lots of new people would like play our product.)
> If it hasn't been already created, could you, please, create an "improvement" issue with a description of morale and luck types, @LeHerosInconnu ?

Done here: https://github.com/ihhub/fheroes2/issues/2808.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
